roles/common/postgres/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install postgresql
+  apt:
+    name: postgresql
+    state: present
+    update_cache: yes
+
+- name: Install psycopg2
+  pip:
+    name: psycopg2
+    version: 2.7.3.2
+    state: present
+
+- name: Create postgres role for custom user
+  become: true
+  become_user: postgres
+  postgresql_user:
+    name: "{{ custom_user }}"
+
+- name: Create postgres custom database
+  become: true
+  become_user: postgres
+  postgresql_db:
+    name: "{{ custom_database }}"
```

Comment:
Suggest adding a `_name` suffix to the variable to better clarify that it's the name of the database.

Reply:
done